debsources/app/copyright/views.py

```diff
@@ -122,12 +123,34 @@ def _license_of_files(self, checksum, package, suite):
                                                              f['version'])))
         return result
 
+    def batch_api(self, checksums, package=None, suite=None):
+        results = []
+        for sha in checksums:
+            files = self._get_files(sha, package, suite)
+            if len(files) is 0:
```

Comment:
On Tue, Jun 23, 2015 at 05:05:31AM -0700, Matthieu Caneill wrote:
> > +            if len(files) is 0:
> 
> any reason to use 'is' instead of '==' here?

Even better, use "if not files:", which would be equivalent and way more
pythonic. (Assuming that "files" is / behaves like a list, that is.)

Cheers.
-- 
Stefano Zacchiroli  . . . . . . .  zack@upsilon.cc . . . . o . . . o . o
Maître de conférences . . . . . http://upsilon.cc/zack . . . o . . . o o
Former Debian Project Leader . . . . . @zacchiro . . . . o o o . . . o .
« the first rule of tautology club is the first rule of tautology club »